openwisp_radius/tasks.py

```diff
@@ -39,3 +49,38 @@ def delete_unverified_users(older_than_days=1, exclude_methods=''):
 @shared_task
 def convert_called_station_id(unique_id=None):
     management.call_command('convert_called_station_id', unique_id=unique_id)
+
+
+@shared_task
+def send_login_email(accounting_data):
+    User = get_user_model()
+    username = accounting_data.get('username', None)
+    org_uuid = accounting_data.get('organization')
+    user = User.objects.filter(username=username).first()
```

Comment:
I think a try/except here using `get()` instead of filter would result more readable and with less indented code, we'll have to test the case in which no user corresponds to the radius accounting anyway because it can happen when using realms or when using radius checks